### DISCUSSION TITLE
Account is blocked

### DISCUSSION THREAD

**Discussion by kevdevlife** [vercel/community#3529] [Help]
### Summary

Hey My account is blocked, i've deleted all the apps on this account, and running 2 premium accounts on the side...

### Steps to Reproduce

Unblock please, i can't deploy on one of my other teams right now.

**Comment by quarkhadron** [comment]
I'm having the same issue sir. 
https://github.com/orgs/vercel/discussions/3427

Been a week and I haven't got a solution yet.

@amyegan Could you please help us?

**Comment by amyegan** [comment]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
